packages/core/getBusyTimes.ts

```diff
@@ -124,6 +125,18 @@ export async function getBusyTimes(params: {
                 },
               },
             },
+            {
+              startTime: { lte: endTimeDate },
+              endTime: { gte: startTimeDate },
+              eventType: {
+                id: eventTypeId,
+                requiresConfirmation: true,
```

Comment:
I'm wondering if we should do this in a filter; it depends on how much of a performance hit this is